lib/msal-common/src/utils/Constants.ts

```diff
@@ -3,92 +3,88 @@
  * Licensed under the MIT License.
  */
 
-export const Constants = {
-    LIBRARY_NAME: "MSAL.JS",
-    SKU: "msal.js.common",
-    // Prefix for all library cache entries
-    CACHE_PREFIX: "msal",
-    // default authority
-    DEFAULT_AUTHORITY: "https://login.microsoftonline.com/common/",
-    DEFAULT_AUTHORITY_HOST: "login.microsoftonline.com",
-    DEFAULT_COMMON_TENANT: "common",
-    // ADFS String
-    ADFS: "adfs",
-    DSTS: "dstsv2",
-    // Default AAD Instance Discovery Endpoint
-    AAD_INSTANCE_DISCOVERY_ENDPT:
-        "https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=",
-    // CIAM URL
-    CIAM_AUTH_URL: ".ciamlogin.com",
-    AAD_TENANT_DOMAIN_SUFFIX: ".onmicrosoft.com",
-    // Resource delimiter - used for certain cache entries
-    RESOURCE_DELIM: "|",
-    // Placeholder for non-existent account ids/objects
-    NO_ACCOUNT: "NO_ACCOUNT",
-    // Claims
-    CLAIMS: "claims",
-    // Consumer UTID
-    CONSUMER_UTID: "9188040d-6c67-4c5b-b112-36a304b66dad",
-    // Default scopes
-    OPENID_SCOPE: "openid",
-    PROFILE_SCOPE: "profile",
-    OFFLINE_ACCESS_SCOPE: "offline_access",
-    EMAIL_SCOPE: "email",
-    CODE_GRANT_TYPE: "authorization_code",
-    RT_GRANT_TYPE: "refresh_token",
-    S256_CODE_CHALLENGE_METHOD: "S256",
-    URL_FORM_CONTENT_TYPE: "application/x-www-form-urlencoded;charset=utf-8",
-    AUTHORIZATION_PENDING: "authorization_pending",
-    NOT_DEFINED: "not_defined",
-    EMPTY_STRING: "",
-    NOT_APPLICABLE: "N/A",
-    NOT_AVAILABLE: "Not Available",
-    FORWARD_SLASH: "/",
-    IMDS_ENDPOINT: "http://169.254.169.254/metadata/instance/compute/location",
-    IMDS_VERSION: "2020-06-01",
-    IMDS_TIMEOUT: 2000,
-    AZURE_REGION_AUTO_DISCOVER_FLAG: "TryAutoDetect",
-    REGIONAL_AUTH_PUBLIC_CLOUD_SUFFIX: "login.microsoft.com",
-    KNOWN_PUBLIC_CLOUDS: [
-        "login.microsoftonline.com",
-        "login.windows.net",
-        "login.microsoft.com",
-        "sts.windows.net",
-    ],
-    SHR_NONCE_VALIDITY: 240,
-    INVALID_INSTANCE: "invalid_instance",
-};
-
-export const HttpStatus = {
-    SUCCESS: 200,
-    SUCCESS_RANGE_START: 200,
-    SUCCESS_RANGE_END: 299,
-    REDIRECT: 302,
-    CLIENT_ERROR: 400,
-    CLIENT_ERROR_RANGE_START: 400,
-    BAD_REQUEST: 400,
-    UNAUTHORIZED: 401,
-    NOT_FOUND: 404,
-    REQUEST_TIMEOUT: 408,
-    GONE: 410,
-    TOO_MANY_REQUESTS: 429,
-    CLIENT_ERROR_RANGE_END: 499,
-    SERVER_ERROR: 500,
-    SERVER_ERROR_RANGE_START: 500,
-    SERVICE_UNAVAILABLE: 503,
-    GATEWAY_TIMEOUT: 504,
-    SERVER_ERROR_RANGE_END: 599,
-    MULTI_SIDED_ERROR: 600,
-} as const;
-export type HttpStatus = (typeof HttpStatus)[keyof typeof HttpStatus];
+export const LIBRARY_NAME = "MSAL.JS";
```

Comment:
Can we evaluate each of these to see if they're still referenced anywhere and remove if not used? This looks like an example of something we don't use anymore, but worth double checking.

Reply:
Removed unused constants